tests/Unit/Services/UpdateAddressTest.php

```
@@ -0,0 +1,105 @@
+        $this->assertEquals('NY', $updatedAddress->state);
+        $this->assertEquals('10014', $updatedAddress->postal_code);
+        $this->assertEquals('United States', $updatedAddress->country);
+        $this->assertFalse($updatedAddress->is_active);
+
+        Queue::assertPushedOn(
+            queue: 'low',
+            job: UpdateUserLastActivityDate::class,
+            callback: function (UpdateUserLastActivityDate $job) use ($user): bool {
+                return $job->user->id === $user->id;
+            },
+        );
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Queue' in method 'it_updates_an_address_with_all_fields'. (Clean Code Rules)

(StaticAccess)